Fst2List.cpp

```diff
@@ -40,30 +42,31 @@ namespace unitex {
 const char
     * usage_Fst2List =
         "Usage:\n"
-          "Fst2List [-o outFile][-p s/f/d][-[a/t] s/m] [-m] [-d] [-f s/a][-s[0s] \"Str\"] [-r[s/l] \"Str\"] [-l line#] [-i subname]* [-c SS=0xxxx]* fname\n"
-          " fname : name of the input file name with extension \".fst2\"\r\n"
-          " -S : write path on standard output\r\n"
-          " -o outFile : if this option and -S not exist, save paths at \"file\"lst.txt\r\n"
-          " -[a/t] s/m : mode de automata or transducteur, s=single initial, m = multi-inital\r\n"
+          "Fst2List [-o outFile][-p s/f/d][-a/t s/m] [-m] [-d] [-f s/a] [--io-separator \"Str\"] [--stop-mark \"Str\"][-s \"Str\"] [-r s/l/x \"Str\"] [-l line#] [-i subname]* [-c SS=0xxxx]* fname\n"
+          " fname: input file name with extension \".fst2\"\r\n"
+          " -S: write path on standard output\r\n"
+          " -o outFile: if this option and -S are not used, save paths in \"file\"lst.txt\r\n"
+          " -a/t s/m, --automaton/transducer s/m: a = write outputs, t = ignore outputs, \r\n"
```

Comment:
The Users Manual have a better description of this option:
`-[a/t] s/m : indicates if the program must take into account (t) or not (a)
the outputs of the grammars if any. s indicates that there is only one initial
state, whereas m indicates that there are several ones (this mode is useful in
Korean). The default value is -a s`

Reply:
Wouldn't brackets mean `a/t` is optional and `s/m` can be used separatly ?

Reply:
Good point ! The adopted convention is that `/` indicates mutually exclusive information (e.g see [here](https://github.com/UnitexGramLab/unitex-core/blob/master/SelectOutput.cpp#L43)) and `[]` identifies an optional argument. 

To avoid confusions, some time ago, I started to adopt a [doctop-based](http://docopt.org/) syntax. For now, only `Compress` has been migrated (see [here](https://github.com/UnitexGramLab/unitex-core/blob/23cf3926fd6d1f532e07d74fa2f08fd7107054f2/Compress.cpp#L47)). 

In the case of `Fst2List`, feel free to use `()` to indicate that one of the arguments is required and `|` instead of '/' for mutually exclusive arguments, i.e. `[(-a|-t) [s|m]]`